### PR TITLE
Calculate rental cost

### PR DESCRIPTION
**Pull request by rvrazovskyi**
Transportation on vacation

### CODE REVIEW

**Review by PolinaRomanenko — COMMENTED** [review]
You need to solve this task - I row of code wouldn't solve it, I'm sorre

**Comment by PolinaRomanenko** [comment]
Also, at first - make changes in your own - cloned repository - not it the mate's repository

**Comment by rvrazovskyi** [comment]
Sorry. Have a problems I delete that repository,make new one with branch develop and send again   Solution #2808

**Comment by Kate-Solovyova** [comment]
You should create a pull request from your develop branch. Read again [guideline](https://github.com/mate-academy/js_task-guideline/blob/master/README.md)

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
```
[the remark this answers is not in the document]
Comment (rvrazovskyi) [reply src/calculateRentalCost.js:30]: Sorry.I have a problem and delete that repository, make a new one with branch develop, and send again Solution https://github.com/mate-academy/js_task-transportation-on-vacation/pull/2808

[the remark this answers is not in the document]
Comment (rvrazovskyi) [reply src/calculateRentalCost.js:30]: Sorry.I have a problem and delete that repository, make a new one with branch develop, and send again Solution https://github.com/mate-academy/js_task-transportation-on-vacation/pull/2808

**Comment by rvrazovskyi** [comment]
Hi, I make a new one with branch develop, and send again Solution https://github.com/mate-academy/js_task-transportation-on-vacation/pull/2808

**Review by 4ksandrey — COMMENTED** [review]
Next time be careful with sending tasks, aproove your Solution 2808

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
-  // write code here
+  const PRICE_PER_DAY = 40;
 }
 
 module.exports = calculateRentalCost;
```
Comment (4ksandrey) [inline src/calculateRentalCost.js:34]: Update link as per guideline

**Comment by rvrazovskyi** [comment]
I'm sorry, I can't understand how to insert a link and where. How to fix it, I'm confused.

File: src/calculateRentalCost.js (hunk 1 above)
Comment (VeronikaSynieokaia) [inline src/calculateRentalCost.js:30]: The function calculateRentalCost does not have a return statement. Make sure to calculate the rental cost based on the given days and return the result.

**Comment by VeronikaSynieokaia** [comment]
fixes are required

**Pull request closed without merging**